File: script/convenience/yakusu.c
#include <stdio.h>

void main(int argc, char** argv) {
  int i, j;

  if (argc != 3) {
    printf(" usage : %s [value] [#.yakusu]\n", argv[0]);
    return;
  }
  int valu = atoi(argv[1]);
  int Nyakusu = atoi(argv[2]);

  int tmp_j = 2;

  for (  i=0;     i<=Nyakusu; i++)
    for (j=tmp_j; j<valu;    j++) {
      if (valu%j == 0) {
	printf(" %d", j);
	tmp_j = j+1;
	break;
      }
    }
  printf("\n");
}

File: scripts/convenience/yakusu.py
#!/usr/bin/python

if __name__ == "__main__":
    import sys
    import math
    
    argv = sys.argv
    argc = len(argv)
    
    if (argc != 3):
        print "usage : %s [value] [#.yakusu]" % argv[0]
        quit()
    
    valu    = int(argv[1])
    Nyakusu = int(argv[2])
    
    tmp_j   = 2
    
    for i in range(Nyakusu+1):
        for j in range(tmp_j, valu):
            if (valu%j == 0):
                print " %d" % j,
                tmp_j = j+1
                break
    print
